handlers/svg-open-street-map/osm-svg.py

```diff
@@ -0,0 +1,216 @@
+
+    if ("ca.mcgill.a11y.image.capability.DebugMode"
+        not in contents['capabilities']
+            or "ca.mcgill.a11y.image.renderer.SVGLayers"
+            not in contents["renderers"]):
+        logging.debug("Debug mode inactive")
+        print("debug inactive")
+        response = {
+            "request_uuid": contents["request_uuid"],
+            "timestamp": int(time.time()),
+            "renderings": []
+        }
+        try:
+            validator = jsonschema.Draft7Validator(
+                response_schema, resolver=resolver)
+            validator.validate(response)
+        except jsonschema.exceptions.ValidationError as error:
+            logging.error(error)
+            return jsonify("Invalid Preprocessor JSON format"), 500
+        logging.debug("Sending response")
+        return response
+
+    if "ca.mcgill.a11y.image.preprocessor.openstreetmap"\
+            not in preprocessor:
+        logging.info("Not for OSM preprocessor. Skipping ...")
+        return "", 204
+    dimensions = 500, 500
+    svg = draw.Drawing(
+        dimensions[0],
+        dimensions[1])
+
+    svg_layers = []
+    data = preprocessor["ca.mcgill.a11y.image.preprocessor.openstreetmap"]
+    if "streets" in data:
+        streets = data["streets"]
+        lat = data["bounds"]["latitude"]
+        lon = data["bounds"]["longitude"]
+        lon_min = lon["min"]
+        lat_min = lat["min"]
+        lon_max = lon["max"]
+        lat_max = lat["max"]
+        # Scale the lon/lat units to svg pixels equivalent.
+        scaled_longitude = dimensions[0] / (lon_max - lon_min)
+        scaled_latitude = dimensions[1] / (lat_max - lat_min)
+        bounds = [[lon_min, lat_min],
+                  [lon_min, lat_max],
+                  [lon_max, lat_max],
+                  [lon_max, lat_min],
+                  [lon_min, lat_min]]
+        # Draw bounding box for the streets
+        for index in range(len(bounds) - 1):
+            p = draw.Path(stroke="orange", stroke_width=6, fill='none')
+            p.M(scaled_longitude * (bounds[index][0] - lon_min),
+                scaled_latitude * (bounds[index][1] - lat_min))
+            p.L(scaled_longitude * (bounds[index + 1][0] - lon_min),
+                scaled_latitude * (bounds[index + 1][1] - lat_min))
+            svg.append(p)
+
+        colors = [
+            "red",
+            "blue",
+            "springgreen",
+            "deeppink",
+            "orange",
+            "purple",
+            "cyan",
+            "coral",
+            "teal",
+            "indigo",
+            "lime",
+            "chocolate",
+            "magenta",
+            "crimson",
+            "deepskyblue",
+            "greenyellow",
+            "gold",
+            "green",
+            "aqua",
+            "navy",
+            "royalblue",
+            "forestgreen",
+            "dodgerblue"
+        ]
+        # Draw the streets with svg.
+        for street in range(len(streets)):
```

Comment:
It is really difficult to read if you are using a variable named "street" as an index when its name follows convention for an iterator.